tests/skyserve/update/new_autoscaler_before.yaml.j2

```diff
@@ -5,10 +5,10 @@ service:
   replicas: 2
 
 resources:
-  cloud: gcp
+  cloud: {{generic_cloud}}
   ports: 8081
   cpus: 2+
 
 workdir: examples/serve/http_server
 
-run: python3 server.py
+run: python3 server.py --port 8081
```

Comment:
Good catch! Is the other changes in this PR needed except for this one?

Reply:
The test will work only with this change.